results/cmd/api/main_test.go

```diff
@@ -186,8 +432,92 @@ func TestDeleteTaskRun(t *testing.T) {
 		t.Fatalf("failed to delete taskrun: %v", r.String())
 	}
 
-	// Check if a deleted taskrun can be delete again
+	// Check if a deleted taskrun can be deleted again
 	if _, err := srv.DeleteTaskRunResult(ctx, &pb.DeleteTaskRunRequest{ResultsId: r.GetResultsId()}); status.Code(err) != codes.NotFound {
 		t.Fatalf("same taskrun not supposed to be deleted again: %v", r.String())
 	}
 }
+func TestListTaskRuns(t *testing.T) {
+	// Create a temporary database
+	srv, err := setupTestDB(t)
+	if err != nil {
+		t.Fatalf("failed to setup db: %v", err)
+	}
+
+	// Connect to fake server and create a new taskrun
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
Go tests have a timeout flag that can be configured, so we likely don't need to do this unless we have a particular reason to have a shorter per-request timeout.

Reply:
done